cmd/kubelet/app/server.go

```diff
@@ -345,6 +350,8 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.KubeAPIBurst, "kube-api-burst", s.KubeAPIBurst, "Burst to use while talking with kubernetes apiserver")
 	fs.BoolVar(&s.SerializeImagePulls, "serialize-image-pulls", s.SerializeImagePulls, "Pull images one at a time. We recommend *not* changing the default value on nodes that run docker daemon with version < 1.9 or an Aufs storage backend. Issue #10959 has more details. [default=true]")
 	fs.BoolVar(&s.ExperimentalFlannelOverlay, "experimental-flannel-overlay", s.ExperimentalFlannelOverlay, "Experimental support for starting the kubelet with the default overlay network (flannel). Assumes flanneld is already running in client mode. [default=false]")
+	fs.Var(&s.SystemReserved, "system-reserved", "A set of ResourceName=ResourceQuantity pairs that describe resources reserved for non-kubernetes components. Currently only cpu and memory are supported. [default=none]")
```

Comment:
Yeah. Any solution that will help our users is fine by me.

On Wed, Dec 16, 2015 at 5:30 PM, Tim St. Clair <notifications@github.com>
wrote:

> In cmd/kubelet/app/server.go
> <https://github.com/kubernetes/kubernetes/pull/18611#discussion_r47859517>
> :
>
> > @@ -345,6 +350,8 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
> >  	fs.IntVar(&s.KubeAPIBurst, "kube-api-burst", s.KubeAPIBurst, "Burst to use while talking with kubernetes apiserver")
> >  	fs.BoolVar(&s.SerializeImagePulls, "serialize-image-pulls", s.SerializeImagePulls, "Pull images one at a time. We recommend *not* changing the default value on nodes that run docker daemon with version < 1.9 or an Aufs storage backend. Issue #10959 has more details. [default=true]")
> >  	fs.BoolVar(&s.ExperimentalFlannelOverlay, "experimental-flannel-overlay", s.ExperimentalFlannelOverlay, "Experimental support for starting the kubelet with the default overlay network (flannel). Assumes flanneld is already running in client mode. [default=false]")
> > +	fs.Var(&s.SystemReserved, "system-reserved", "A set of ResourceName=ResourceQuantity pairs that describe resources reserved for non-kubernetes components. Currently only cpu and memory are supported. [default=none]")
>
> Is this the right place to describe that? Would you accept a link to the
> API documentation for resource list instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/18611/files#r47859517>.
>